README.md

```diff
@@ -57,7 +48,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: this is O(n^2) because there are two nested .each loops.
```

Comment:
Got it! Thanks -- looks like I screwed up the formatting with markdown. I think you got what I meant but I was going for (n\*m) and (n\*\*2). Thanks Crystal!